services/movie_session.py

```diff
@@ -1,21 +1,24 @@
 from django.db.models import QuerySet
-from typing import Optional
 
 
 from db.models import MovieSession, Movie, CinemaHall
 
 
-def create_movie_session(movie_show_time: str,
-                         movie_id: int,
-                         cinema_hall_id: int) -> None:
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
     movie = Movie.objects.get(pk=movie_id)
     hall = CinemaHall.objects.get(pk=cinema_hall_id)
-    MovieSession.objects.create(show_time=movie_show_time,
-                                cinema_hall=hall,
-                                movie=movie)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=hall,
+        movie=movie
+    )
```

Comment:
Still not fixed. In order to not make 2 additional sql queries for movie and hall, you can set movie_id and cinema_hall_id directly. You can do it despite you haven't these fields in your model because Django is developed in the way that in the code you can use your added fields(movie, cinema_hall) and they contain related objects but under the hood in database ID is saved and column name is movie_id and cinema_hall_id because database doesn't know what is Django, for is related object but knows what is id